libs/SalesforceSDKCore/Configuration/SalesforceSDKCore-Common-Static.xcconfig

```diff
@@ -10,5 +10,3 @@
 
 INSTALL_PATH = /
 SKIP_INSTALL = YES
-
-HEADER_SEARCH_PATHS = $(BUILT_PRODUCTS_DIR)/Headers $(TARGET_BUILD_DIR)/Headers $(inherited)
```

Comment:
Looks like SalesforceSDKCoreStatic is otherwise picking up its necessary `HEADER_SEARCH_PATHS` from Packaging-Static.xcconfig, so this one's fine to ditch.